schema/schema.go

```diff
@@ -55,7 +49,21 @@ var searchDataImport = `{
         {"name": "previous_date", "type": "string", "default": ""}
       ]
     }}},
-    {"name": "provisional_date", "type": "string", "default": ""},
+    {"name": "cancelled", "type": "boolean", "default": false},
```

Comment:
The new order of these in this service does not match the order in the `dp-search-data-importer` (https://github.com/ONSdigital/dp-search-data-importer/pull/84/files#diff-eafe62eae191e88faad4c223c4de9af268c175ee68695872e175b814d7c5baa5R40)
Note the most important consistency is between the schema files in different apps.